packages/shopify-api/adapters/__e2etests__/test_apps/test-web-api-app.ts

```diff
@@ -21,9 +22,12 @@ const RED = '\x1b[31m';
 const GREEN = '\x1b[32m';
 const RESET = '\x1b[39m';
 
-const HttpClient = httpClientClass(config, 'http');
+const RestClient = restClientClass({
```

Comment:
I think a better refactoring here would be to use `abstractFetch` instead, but a lot of these tests are actually testing error handling from the client, which would have to be dropped. Not sure what the best approach is here.